tests/src/test/java/alluxio/client/fuse/FuseFileSystemIntegrationTest.java

```diff
@@ -209,6 +212,25 @@ public void head() throws Exception {
     Assert.assertEquals("Alluxio Head Test\n", result);
   }
 
+  @Test
+  public void ls() throws Exception {
+    // ls -sh has different results in osx
+    Assume.assumeTrue(OSUtils.isLinux());
+    String testFile = "/lsTestFile";
+    ShellUtils.execCommand("dd", "if=/dev/zero",
+        "of=" + sMountPoint + testFile, "count=10", "bs=" + 4 * Constants.MB);
+
+    // Fuse release() is async, Fuse getattr() will wait for file completed and get the
+    // right file size
+    String out = ShellUtils.execCommand("ls", "-sh", sMountPoint + testFile);
+    Assert.assertFalse(out.isEmpty());
+    Assert.assertEquals("40M", out.split("\\s+")[0]);
+
+    Assert.assertTrue(sFileSystem.exists(new AlluxioURI(testFile)));
+    Assert.assertEquals(40 * Constants.MB,
+        sFileSystem.getStatus(new AlluxioURI(testFile)).getLength());
```

Comment:
Is creating a 40MB file necessary? It might be a bit heavy weight for running a test?

Reply:
Good catch! I changed to 40KB. Our block size is 4K, 40MB file will take a lot of blocks. Thanks